pkg/controller/common.go

```diff
@@ -78,6 +81,33 @@ func (c *Controller) GetAndFilter(ctx context.Context, key client.ObjectKey, obj
 	return true, nil
 }
 
+func (c *Controller) AreNetworkPoliciesEnabled(ctx context.Context, resource resources.Resource) (bool, error) {
+	for name, value := range resource.GetAnnotations() {
+		if name == harbormetav1.NetworkPoliciesAnnotationName {
+			return value == harbormetav1.NetworkPoliciesAnnotationEnabled, nil
+		}
+	}
+
+	defaultNetworkPoliciesStatusItem, err := configstore.Filter().Store(c.ConfigStore).Slice(config.NetworkPoliciesStatusKey).GetFirstItem()
+	if err != nil {
+		return false, err
+	}
+
+	defaultNetworkPoliciesStatus, err := defaultNetworkPoliciesStatusItem.Value()
+	if err != nil {
+		return false, err
+	}
+
+	return defaultNetworkPoliciesStatus == harbormetav1.NetworkPoliciesAnnotationEnabled, nil
+}
+
+func (c *Controller) DefaultAnnotations() map[string]string {
```

Comment:
As talked together, remove or rename this functions, or call it everywhere there is resources with annotations.